todolist/settings.py

```diff
@@ -65,8 +65,8 @@
         'ENGINE': 'mysql.connector.django',
         'NAME': 'app_db',
         'USER': 'app_user',
-        'PASSWORD': '1234',
-        'HOST': '172.17.0.2',  # You can use a different host if your MySQL server is on a remote machine.
+        'PASSWORD': '1q2q3q4',
```

Comment:
The database password is hardcoded, which is a security risk. Consider using environment variables to manage sensitive information securely.